packages/next/src/build/webpack/loaders/next-app-loader.ts

```diff
@@ -116,7 +116,7 @@ async function createTreeCodeFromPath(
         continue
       }
 
-      const parallelSegmentPath = segmentPath + '/' + parallelSegment
+      const parallelSegmentPath = path.join(segmentPath, parallelSegment)
```

Comment:
I think our segments are just using `/`? They're not file paths though

Reply:
i was seeing duplicated slashes in between before, but seems gone now. reverted